### ISSUE TITLE
LOI/LRI ACKs MVP

### ISSUE THREAD

**Issue by brandonnava** [CDCgov/prime-reportstream#16393]
### User Story
As a ReportStream ETOR sender I want message confirmation ACKs sent to me following industry standard LOI/LRI specification to ensure the timely delivery of my data

### Description/Use Case
A number of pilot partners have asked us if we support ACK messaging and some have even specified if it follows the ACK implementation following the LOI/LRI spec. The CDC has noticed the need as well as wants to support more industry standardization so has asked us as part of their annual goals to implement the ACKs component of the LOI/LRI spec. Eventually they or our sender partners will probably want the full ACK implementation but for now have agreed to an MVP that just implements the Basic Synchronous Accept ACKs for our Submission API following the needs of our ETOR partner Flexion. Their exact requirements are provided in the dev notes.  

### Risks/Impacts/Considerations
Risks: piloting the MVP will require an engaged partner willing to setup the API connection and debug the response. Currently that partner is Flexion and they seem engaged. They also have to agree with us on the contents of the ACK message and nail down requirements in a timely manner for us to test and launch before the quarter is through. 

### Dev Notes
Flexion ETOR ACK Requirements: https://cdc.sharepoint.com/:w:/r/teams/OLSS_ETOR_ReportStream/Shared%20Documents/ReportStream%20ETOR%20ACK%20Requirements.docx?d=we5a9fae68cd24c99a757f027562c1737&csf=1&web=1&e=x3lgDZ
HL7 over HTTP spec: http://v2plus.hl7.org/2021Jan/hl7-over-http.html
Epic's Orders & Results LRI ORM spec documentation: https://cdc.sharepoint.com/:b:/t/ReportStream/EchwA7-zxUFArPsluFSbZnIBJe4O9oYwzGjgvy5vAm-tPQ?e=ZP7ji6
Epic's Orders & Results ELR spec documentation:https://cdc.sharepoint.com/:b:/t/ReportStream/ETXO5yfN0NtGqOOKrbrVMpYBibSFFRj_duOXsi9qfjH76w?e=HsnptB

### Acceptance Criteria

- [ ] Flexion can send a test order message and gets back a valid HL7 ACK message
- [ ] The ACK message contains the MSH and MSA fields outlined in the ETOR ACK Requirements doc

**Comment by arnejduranovic** [comment]
@jsutantio No, unless Flexion has issues and needs changes but that can be a different epic/tickets imo.
